### PR TITLE
Add some changes on BoardConfig.mk

### PR DESCRIPTION
**Pull request by Igorpetinik**

### CODE REVIEW

File: BoardConfig.mk [hunk 1]
```diff
@@ -126,8 +126,10 @@ TW_BRIGHTNESS_PATH := "/sys/class/backlight/panel/brightness"
 TW_MAX_BRIGHTNESS := 255
 TW_DEFAULT_BRIGHTNESS := 200
 TW_USE_SAMSUNG_HAPTICS := true
-TW_Y_OFFSET := 80
-TW_H_OFFSET := -80
+
+# Screen Offsets
+# TW_Y_OFFSET := 80
```
Comment (DevZillion) [inline BoardConfig.mk]: Uncomment the screen offsets

File: BoardConfig.mk [hunk 2]
```diff
@@ -146,4 +148,3 @@ TW_MTP_DEVICE := "Galaxy A31"
 
 # Making the recovery.img smaller
 BOARD_HAS_NO_REAL_SDCARD := true
-
```
Comment (DevZillion) [inline BoardConfig.mk:149]: Please avoid removing the blank line at the end of the file :)

**Review by DevZillion — CHANGES_REQUESTED** [review]
ptal

**Review by Igorpetinik — COMMENTED** [review]
Fix

**Pull request closed without merging**